modules/collect/src/main/java/com/opengamma/strata/collect/io/CsvRow.java

```diff
@@ -300,7 +300,7 @@ public String getValue(String header) {
    * This returns the value of the first column where the header matches the specified header pattern.
```

Comment:
In this file, I added two findValue methods supporting post processing as I realised it was almost always the usecase.